client/reader/sidebar/expandable-heading.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import noop from 'lodash/utility/noop';
+
+/**
+ * Internal dependencies
+ */
+import SidebarHeading from 'layout/sidebar/heading';
+import Gridicon from 'components/gridicon';
+import Count from 'components/count';
+
+const ExpandableSidebarHeading = ( { title, count, onClick } ) => (
+	<SidebarHeading onClick={ onClick }>
```

Comment:
Looking at this component I think we could make it a prop on `<SidebarHeading expandable>` and pass `Count` as a child only if you need to.